Tests/NetworkServiceTests/HTTPMethodTests.swift

```diff
@@ -0,0 +1,23 @@
+import NetworkService
+import XCTest
+
+final class HTTPMethodTests: XCTestCase {
+    func testExpectedHTTPMethodsEqualToIncludedInAllCases() {
+        // Given
+        let currentHTTPMethods = Set(HTTPMethod.allCases)
```

Comment:
If we need the `allCases` only in the test target, then please implement the conformance to the protocol in the test target, no in the main target.

Does that make sense?

Reply:
Fixed